src/main/java/com/teamcautionrobotics/robot/ExpanderHatch.java

```diff
@@ -1,14 +1,21 @@
 package com.teamcautionrobotics.robot;
 
+import edu.wpi.first.wpilibj.DoubleSolenoid;
 import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
 
 public class ExpanderHatch {
 
     // pneumatics objects
     // reacher extends away from the robot
     // grabber piston applies friction to the hatch by expanding
-    private final Solenoid reacher;
-    private final Solenoid grabber;
+    private Solenoid reacher;
+    private Solenoid grabber;
+
+    private DoubleSolenoid doubleReacher;
+    private DoubleSolenoid doubleGrabber;
```

Comment:
I might be partial to ordering these reacher, doubleReacher, grabber, doubleGrabber. You can decide which you prefer.

Reply:
I like it how it is.